### PR TITLE
TODO improvement on aten_sym_size op when ORT==1.15 | feat(atenlib)

### PR DESCRIPTION
**Pull request by titaiwangms**
https://github.com/microsoft/onnx-script/pull/484#discussion_r1136105039

Not sure why another file was linted...

### CODE REVIEW

File: onnxscript/backend/onnx_export_test.py [hunk 1]
```diff
@@ -168,7 +168,9 @@ def test_export2python_produces_correct_onnx_script_model(
             onnxruntime_pybind11_state.InvalidArgument,  # pylint: disable=c-extension-no-member
         ) as e:
             self.skipTest(f"Unable to load the model: {e}")
-        except onnxruntime_pybind11_state.RuntimeException as e:  # pylint: disable=c-extension-no-member
+        except (
+            onnxruntime_pybind11_state.RuntimeException
+        ) as e:  # pylint: disable=c-extension-no-member
```
Comment (justinchuby) [inline onnxscript/backend/onnx_export_test.py:173]: You may revert this change. We prob need to bump black. I will send a pr

File: onnxscript/function_libs/torch_aten/ops/core.py [hunk 2]
```diff
@@ -5540,6 +5540,14 @@ def aten_sym_size(self: TReal, dim: int = 0) -> TReal:
     """sym_size(Tensor self, int dim) -> Tensor"""
     # NOTE: onnxscript doesn't support attribute process,
     # so op.Shape(self, start=dim, end=dim + 1) is not supported.
+
+    # TODO(titaiwang): ORT==1.15 fixes SegFault
+    # https://github.com/microsoft/onnx-script/pull/484#discussion_r1136105039
+    # Change the op to:
+    # shape = op.Shape(self)
+    # idx= op.Reshape(dim, [1])
+    # return op.Gather(shape, idx)
```
Comment (justinchuby) [inline onnxscript/function_libs/torch_aten/ops/core.py:5549]: Why gather when we can slice? Just for my learning

**Comment by justinchuby** [comment]
fyi we just decided to change the PR title format following https://github.com/microsoft/onnx-script/issues/577

File: onnxscript/function_libs/torch_aten/ops/core.py (hunk 2 above)
Comment (titaiwangms) [reply onnxscript/function_libs/torch_aten/ops/core.py:5549]: From Rama: As discussed in another PR, dim+1 also doesn't work if dim is -1. Hence, Gather seems a better solution.

**Review by justinchuby — APPROVED** [review]

**Pull request merged**